sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -453,8 +453,8 @@ class Dataset[T] private[sql](
    * Returns true if this [[Dataset]] contains one or more sources that continuously
    * return data as it arrives. A [[Dataset]] that reads data from a streaming source
    * must be executed as a [[ContinuousQuery]] using the `startStream()` method in
-   * [[DataFrameWriter]].  Methods that return a single answer, (e.g., `count()` or
-   * `collect()`) will throw an [[AnalysisException]] when there is a streaming
+   * [[DataFrameWriter]]. Methods that return a single answer, e.g. `count()` or
```

Comment:
I don't think e.g. is ever written outside a parenthetical (it looks wrong to me that way) and a parenthetical isn't preceded by a comma. I can't find any clear reference for that in 60 seconds though. Your change looks at least right-er than before, even if this is super pedantic. Pedantic is good.